src/app/issuer/components/badgeclass-create/badgeclass-create.component.ts

```diff
@@ -99,10 +99,12 @@ export class BadgeClassCreateComponent extends BaseAuthenticatedRoutableComponen
 	onWindowScroll() {
 		var top = window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop || 0;
 
-		if (top > this.badgeImage.componentElem.nativeElement.offsetTop) {
-			this.scrolled = true;
-		} else {
-			this.scrolled = false;
+		if (this.badgeImage) {
```

Comment:
I feel like this whole construction can be simplified a lot:
```ts
this.scrolled = this.badgeImage && top > this.badgeImage.componentElem.nativeElement.offsetTop;
```
Or is it a requirement that `this.scrolled` is not modified when `this.badgeImage` is `null` or `undefined`? Then it could be
```ts
this.scrolled = this.badgeImage ? top > this.badgeImage.componentElem.nativeElement.offsetTop : this.scrolled;
```

Reply:
I updated it.